src/components/views/read/cvForm/Education.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Remove this and try to fix the lint issue. In case you need to disable the rule, it would be better to do so for only the specific line. Disabling it for the whole file is not a good practice.